### ISSUE TITLE
Lunatic Data : A typo: INPUTED instead of INPUTTED

### ISSUE THREAD

**Issue by laurentC35** [InseeFr/Lunatic-Model#112]
There is a typo in the Lunatic Collected data possible field. It should be INPUTTED instead. I believe it's the right time to correct this before the management mode is released.

Link issue : 
- https://github.com/InseeFr/Lunatic/issues/614
- https://github.com/InseeFr/Eno/issues/712

**Comment by dotSlash-Adwitiya** [comment]
I want to work on this, can you please guide me ?

**Comment by nsenave** [comment]
Duplicate of #105

**Comment by nsenave** [comment]
> [!NOTE]
> Should be done in the `dev-eno-v3` branch
>
> A rework of Lunatic-Model has been done to improve the modeling of certain properties and makes things easier in Eno v3. Yet it introduces some (little, but still) breaking changes with current version in `master`.
>
> The `dev-eno-v3` branch can be merged in the master branch when the DDI to Lunatic transormation in Eno v3 is ready for production.

**Comment by nsenave** [comment]
Fresher thoughts: https://github.com/InseeFr/Eno/issues/712#issuecomment-2033692936

TLDR: needs to be done in v2 so to make sure json/xml conversions that are still used still work.

- [x] done in v3+ version
- [ ] done in v2 version
